rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -44,6 +44,22 @@ class HVAC_SYS:
     SYS_13A = "Sys-13a"
     UNMATCHED = "Not_Sys"
 
+    HVAC_SYSTEM_TYPE_DICTIONARY = {
```

Comment:
It might be cleaner to make this a separate value outside of the HVAC_SYS class.